packages/ui/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,72 @@
+import type { ButtonHTMLAttributes } from 'react';
+import { twMerge } from 'tailwind-merge';
+
+import { Icon } from './Icon';
+import { type BaseProps, ButtonSize, ButtonVariant } from './types';
+
+export function Button({
+    variation = 'primary',
+    className,
+    forwardRef,
+    type = 'button',
+    size = 'default',
+    icon,
+    iconPlacement = 'left',
+    isLoading,
+    disabled,
+    children,
+    contentClassName,
+    ...buttonProps
+}: Button.Props) {
+    return (
+        <button
+            ref={forwardRef}
+            type={type}
+            className={twMerge(
+                'rounded flex items-center justify-center border border-transparent bg-transparent',
+                size === 'small' ? `p-3 label-medium` : `p-4 label-large`,
+                variation === 'primary' &&
+                    `
+                bg-[color:var(--prezly-accent-color)] border-[color:var(--prezly-accent-color)] text-[color:var(--prezly-accent-color-button-text)]
```

Comment:
> `bg-[color:var(--prezly-accent-color)]`

I wonder if we can define these as tailwind classes.  E.g. `bg-accent-color`?

This is possible? @oleg-semyonov wdyt?

Reply:
Updated to use css variables in the tailwind config